images/kcp-registrar/register.sh

```diff
@@ -132,21 +199,38 @@ register() {
                 --syncer-image ghcr.io/kcp-dev/kcp/syncer:$KCP_SYNC_TAG \
                 --resources deployments.apps,services,ingresses.networking.k8s.io,pipelines.tekton.dev,pipelineruns.tekton.dev,tasks.tekton.dev,runs.tekton.dev,networkpolicies.networking.k8s.io \
                 >"$syncer_manifest"
-            KUBECONFIG=${DATA_DIR}/credentials/kubeconfig/compute/${kubeconfigs[$i]} kubectl apply --context ${contexts[$i]} -f "$syncer_manifest"
+            KUBECONFIG=${WORKSPACE_DIR}/credentials/kubeconfig/compute/${kubeconfigs[$i]} kubectl apply \
+                --context "${contexts[$i]}" -f "$syncer_manifest"
         fi
     done
 }
 
-prechecks
-kcp_kubeconfig
-if [[ "$(KUBECONFIG=${kcp_kcfg} kubectl kcp workspace current | cut -d\" -f2)" == "$KCP_ORG:$KCP_WORKSPACE" ]]; then
-    printf "Workspace: %s" "$KCP_ORG:$KCP_WORKSPACE"
-else
-    printf "Switching to organization %s\n" "${KCP_ORG}"
-    switch_org
-    printf "Switching to workspace %s\n" "${KCP_WORKSPACE}"
-    switch_ws
+set_kcp_workspace() {
```

Comment:
I am really not convinced by the name of the function and the log message. What the function is doing is applying a set of manifests. These manifests:
- currently are setting up RBAC
- aiming to configure access to a compute pool

Reply:
While at the moment it's only granting access to the APIExport, it can actually configure any resource in the kcp workspace if the manifests are added to the `environment/kcp` directory.

If you have an idea for a better name, share it with me and I'll update it.

Reply:
proposal: configure_compute_ws()